### ISSUE TITLE
Removing or uninstalling VBA-Dictionary... causes an unusable add-in warning/error message

### ISSUE THREAD

**Issue by elGatoEnBotas** [VBA-tools/VBA-Dictionary#42]
I don't own a Mac but one of my clients does & I installed Dictionary class in my project with varying results according to him. After removing the class from my workbook, I am getting this strange error I haven't seen before: **"OP 
![Unusable Add In after removing Dictionary Class](https://github.com/user-attachments/assets/0942038f-651e-4d54-83e9-dd7d85f862ab)
Experimentation 'We can't start this add in because it isn't set properly.'"** How can we remove that? As it appears on all the Excel files I open, not just my .xlsm workbook. I tried deleting the addins folder for excel (https://answers.microsoft.com/en-us/msoffice/forum/all/cant-remove-an-add-on/a268b25f-03a9-4045-892b-95335ab45755). Didn't work.

**Comment by DecimalTurn** [comment]
I don't think this is related to VBA-Dictionary if you are getting this error when opening other Excel files without VBA macros.

This is likely a problem with Excel itself. 
This post is probably related as it mentions "OP Experimentation": https://techcommunity.microsoft.com/t5/excel/new-excel-error-message/m-p/1436117
